frontend/packages/yki/src/components/registration/PublicRegistrationExamSessionDetails.tsx

```diff
@@ -42,17 +45,19 @@ export const PublicRegistrationExamSessionDetails = ({
     getCurrentLang(),
   );
 
-  const examFeeText = ExamSessionUtils.freeRegistrationPossible(
+  const freeRegistrationPossible = ExamSessionUtils.freeRegistrationPossible(
     examSession,
     loggedInSession,
-    attemptsUsed,
-  )
+  );
+
+  const examFeeText = freeRegistrationPossible
     ? isFree === 'YES'
       ? `0 €`
       : isFree === 'NO'
       ? `${examSession.exam_fee} €`
       : `0 ${translateCommon('or')} ${examSession.exam_fee} €`
     : `${examSession.exam_fee} €`;
```

Comment:
Toi muutos on mun mielestä nyt jo parempi, mutta aloin lukemaan tuota ternary-operaatiota ja se musta aika vaikea lukea. Mä purkaisin sen mielummin if-elseksi, niin sitä on jatkossakin helpompi lukea. 

Esim tälleen:

```suggestion
let examFeeText: string;

if (ExamSessionUtils.freeRegistrationPossible(examSession, loggedInSession, attemptsUsed)) {
  // Free registration is possible
  if (isFree === 'YES') {
    examFeeText = '0 €';
  } else if (isFree === 'NO') {
    examFeeText = `${examSession.exam_fee} €`;
  } else {
    examFeeText = `0 ${translateCommon('or')} ${examSession.exam_fee} €`;
  }
} else {
  // Free registration is not possible
  examFeeText = `${examSession.exam_fee} €`;
}
```

Reply:
Otin tästä pääpointin, muotoilin hieman eri tavalla.